### PR TITLE
First draft of quick start doc

### PR DESCRIPTION
**Pull request by kylegc**

### CODE REVIEW

File: docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
 title: "Appsody Quick Start"
 ---
 
+# Appsody Quick Start
+
+## Installing Appsody
+
+Prerequisite: In order to use Appsody, [Docker](https://docs.docker.com/install/) must be installed and running.
+
+On MacOS, run the following commands:
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: MacOS should be macOS

File: docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
 title: "Appsody Quick Start"
 ---
 
+# Appsody Quick Start
+
+## Installing Appsody
+
+Prerequisite: In order to use Appsody, [Docker](https://docs.docker.com/install/) must be installed and running.
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: As you've said it's a prerequisite you only need the following words:
Prerequisite: [Docker](https://docs.docker.com/install/) installed and running.

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
+brew tap appsody/appsody
+brew install appsody
+```
+On other OSs, check out the [complete installation guide]().
+
+## Using Appsody
+
+Creating a new Appsody project is easy! With only a few commands, you will have a containerized development environment running with the stack of your choice!
+
+First, create a new directory for the project and run `appsody init stack` to download the template project.
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: It's confusing to have the words run `appsody init stack` and then in the example `appsody init nodejs-express`.
Can we refer to `appsody init [stack]`? or `appsody init <stack>`?

File: docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -2,6 +2,44 @@
...
+```
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: I think it'd be good to have some wording like, "The following example uses the `nodejs-express` stack:" above this code snippet.

File: docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -2,6 +2,44 @@
...
+mkdir my-project
+cd my-project
+appsody init nodejs-express
+```
+Tip: You can view other available stacks with `appsody list`
+
+Now you have a fully functional project Appsody project. Start the development container.
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: project Appsody project?

File: docs/getting-started/quick-start.md [hunk 6]
```diff
@@ -2,6 +2,44 @@
...
+```
+appsody run
+```
+Great! Now the project is running in a docker container, and the container is linked to the project source code on your local system. For nodejs-express, navigate to http://localhost:3000 to see the output (other stacks may use a different URL).
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: I'd backtick nodejs-express so it stands out and make the url a 'clickable link' so if a user is following this guide they can do that without typing it in by hand.

File: docs/getting-started/quick-start.md [hunk 7]
```diff
@@ -2,6 +2,44 @@
...
+```
+Tip: You can view other available stacks with `appsody list`
+
+Now you have a fully functional project Appsody project. Start the development container.
+```
+appsody run
+```
+Great! Now the project is running in a docker container, and the container is linked to the project source code on your local system. For nodejs-express, navigate to http://localhost:3000 to see the output (other stacks may use a different URL). 
+
+Finally, lets try changing the code. Edit the file `app.js` to output something other than "Hello World!". Upon saving the file, Appsody will pickup the change and automatically update the container. Refresh http://localhost:3000 to see the new message!
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: lets -> let's

File: docs/getting-started/quick-start.md [hunk 8]
```diff
@@ -2,6 +2,44 @@
...
+
+You are ready to continue developing your application. To stop the container, press ctrl-c in the terminal or run `appsody stop`. To enable the debugger, run `appsody debug`.
```
Comment (helenmasters) [inline docs/getting-started/quick-start.md]: ctrl-c -> Ctrl-C

File: docs/getting-started/quick-start.md [hunk 9]
```diff
@@ -2,6 +2,44 @@
...
+## Installing Appsody
+
+Prerequisite: In order to use Appsody, [Docker](https://docs.docker.com/install/) must be installed and running.
+
+On MacOS, run the following commands:
+```
+brew tap appsody/appsody
+brew install appsody
+```
+On other OSs, check out the [complete installation guide]().
```
[the remark this answers is not in the document]
Comment (helenmasters) [reply docs/getting-started/quick-start.md]: Good idea Musa, but can we make it even more generic to "To install Appsody on Windows or Linux, check out the [installation guide]()."

File: docs/getting-started/quick-start.md (hunk 5 above)
Comment (helenmasters) [reply docs/getting-started/quick-start.md]: "Next, start the development container:" would be better on a new line?

**Comment by kylegc** [comment]
@helenmasters due to the private->public repo change, we cannot update this PR. Please approve and merge it, and then we will address changes in a new PR.

File: docs/getting-started/quick-start.md (hunk 5 above)
Comment (kylegc) [reply docs/getting-started/quick-start.md]: I added "Next," but left it on the same line. Let me know what you think.

**Comment by kylegc** [comment]
These comments have been addressed in PR #14

**Pull request merged**